Dockerfile

```diff
@@ -62,6 +62,8 @@ RUN set -ex \
     && pip install pyasn1 \
     && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql]==$AIRFLOW_VERSION \
     && pip install 'celery[redis]>=4.1.1,<4.2.0' \
+    && pip install flask_bcrypt \
```

Comment:
Might be clearer to add the `password` extras group: `apache-airflow[password]`. We can add this on line 63, or configure via build args if we merge #232.